.gitignore

```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
```

Comment:
Do we need tern?
What is it used for?

Reply:
It's probably not necessary since it's used to enhance editor support.